DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_api.py

```diff
@@ -4,32 +4,51 @@
 
 
 class api:
-    
-    url = "https://api.searchlight.app/v1/"                 #main url for api
 
-    """ constructer initializes the DS creds """
-    def __init__(self, id, key, secret):
+    def __init__(self, id, key, secret, url):
+        """ 
+            constructer initializes the DS creds and creates passkey
+        """
+
+        self.url = "https://" + str(url) + "/"
```

Comment:
Use https://docs.python.org/3/library/urllib.parse.html. To make handle the case, where user might input the server url with scheme (https)